### ISSUE TITLE
Background color on the AI Assistant block doesn't have enough contrast when used on a dark background

### ISSUE THREAD

**Issue by mjaragon** [Automattic/jetpack#34623]
When using the AI Assistant block on a page that has a dark background, it becomes difficult to read the prompt as the user is typing it: 

In the example below, the site has a black background with white text for all its pages:

This is how it looks when the prompt is NOT being edited: 
![image](https://github.com/Automattic/jetpack/assets/30312698/3414445b-aa55-4912-8041-832486aee510)

This is how the text looks when typing the prompt: 
![image](https://github.com/Automattic/jetpack/assets/30312698/2b3d9985-3db0-43e3-b9a5-d00b771d871d)

Steps to reproduce: 

1. Edit or create a page with a dark background
2. Add the AI Assistant block
3. Start typing in the prompt field

**Comment by lhkowalski** [comment]
With the launch of the UI refactor on the assistant (p1HpG7-qaj-p2), I think this issue is now solved. The assistant will stay white as well as the prompt field:

![Image](https://github.com/Automattic/jetpack/assets/6760046/830594cf-fdaf-408b-bf18-1cff12d5db6b)

**Comment by lhkowalski** [comment]
I am closing this as done as a result of the new UI launch.
